### PR TITLE
fix(masthead): chevron size and margin

### PR DESCRIPTION
**Pull request by guilhermelMoraes**
### Related Ticket(s)

[Masthead] L0 options with dropdown is missing 8px space between label & chevron #5650

### CODE REVIEW

File: packages/utilities/src/utilities/settings/settings.js
Comment (jeffchew) [inline packages/utilities/src/utilities/settings/settings.js:14]:
```suggestion
 * Carbon for IBM.com v1.19.0',
```

Sorry about the confusion with two release branches opened. Switching this to the `v1.19.0` branch.

File: packages/utilities/src/utilities/settings/settings.js
Comment (jeffchew) [inline packages/utilities/src/utilities/settings/settings.js:18]:
```suggestion
  version: 'Carbon for IBM.com v1.19.0',
```

Sorry about the confusion with two release branches opened. Switching this to the `v1.19.0` branch.

**Review by jeffchew — APPROVED** [review]
LGTM! Waiting for design approval.

**Review by shixiedesign — COMMENTED** [review]
Almost! Margin is fixed but chevron should be 20px!

image

**Review by shixiedesign — APPROVED** [review]
Looks good now! Thank you!

**Pull request merged**
